src/Gridwich.SagaParticipants.Publication.MediaServicesV3/tests/MediaServicesV3PublicationServiceTests.cs

```diff
@@ -606,8 +606,8 @@ public async void MediaServicesLocatorListPathsFailureTest()
             var amsStreamingPolService = Mock.Of<IMediaServicesV3CustomStreamingPolicyService>();
             var settingsProvider = Mock.Of<ISettingsProvider>();
             string assetName = "myassetname";
-            var amsAccount = new MediaService(storageAccounts: new List<StorageAccount>() { new StorageAccount() { Id = MediaServicesV3PublicationTestData.DefaultStorageId } });
-            var streamingEndpoint = new StreamingEndpoint(name: "default", hostName: "test.azure.net", scaleUnits: 0);
+            var amsAccount = new MediaService(location: "westus", storageAccounts: new List<StorageAccount>() { new StorageAccount() { Id = MediaServicesV3PublicationTestData.DefaultStorageId } });
```

Comment:
Why location is hard coded "westus"?

Reply:
The AMS v6 SDK requests a location parameters when creating a media services account. It is hard coded but has no impact as this is for testing and the resources is not created in real.